proxy/logging/LogField.cc

```diff
@@ -772,7 +772,9 @@ LogFieldList::marshal_len(LogAccess *lad)
   int bytes = 0;
   for (LogField *f = first(); f; f = next(f)) {
     if (f->type() != LogField::sINT) {
-      bytes += f->marshal_len(lad);
+      const int len = f->marshal_len(lad);
```

Comment:
I'm not sure what you're suggesting, Bryan, is different from the `ink_release_assert` on line 776 here. `len` shouldn't just be greater than zero, it should be at least `INK_MIN_ALIGN`.